geomstats/learning/riemannian_mean_shift.py

```diff
@@ -193,3 +193,21 @@ def predict(self, points):
             closest_centers.append(self.centers[closest_center, :])
 
         return gs.array(closest_centers)
+
+    def predict_labels(self, points):
```

Comment:
A unit-test should be added for this function in tests/test_riemannian_mean_shift.py:
https://github.com/geomstats/geomstats/blob/master/tests/tests_geomstats/test_riemannian_mean_shift.py

Reply:
Sure this makes sense